rdrf/rdrf/forms/navigation/quick_links.py

```diff
@@ -402,27 +360,28 @@ def _build_menu(self):
                 **Links.OTHER,
             }
             # menu with everything, used for the admin page
-            if settings.DESIGN_MODE:
+            if is_design_mode():
                 design_menus.update(proms_menus)
-                MenuConfig().all = design_menus
+                self.menu_config.all = design_menus
             else:
-                MenuConfig().all = proms_menus
+                self.menu_config.all = proms_menus
 
     def _group_links(self, group):
         # map RDRF user groups to quick links menu sets
         switcher = {
-            RDRF_GROUPS.PATIENT: MenuConfig().patient,
-            RDRF_GROUPS.PARENT: MenuConfig().parent,
-            RDRF_GROUPS.CLINICAL: MenuConfig().clinical,
-            RDRF_GROUPS.GENETIC_STAFF: MenuConfig().genetic_staff,
-            RDRF_GROUPS.GENETIC_CURATOR: MenuConfig().genetic_curator,
-            RDRF_GROUPS.WORKING_GROUP_STAFF: MenuConfig().working_group_staff,
-            RDRF_GROUPS.WORKING_GROUP_CURATOR: MenuConfig().working_group_curator,
-            RDRF_GROUPS.SUPER_USER: MenuConfig().super_user,
+            RDRF_GROUPS.PATIENT: self.menu_config.patient,
+            RDRF_GROUPS.PARENT: self.menu_config.parent,
+            RDRF_GROUPS.CLINICAL: self.menu_config.clinical,
+            RDRF_GROUPS.GENETIC_STAFF: self.menu_config.genetic_staff,
+            RDRF_GROUPS.GENETIC_CURATOR: self.menu_config.genetic_curator,
+            RDRF_GROUPS.WORKING_GROUP_STAFF: self.menu_config.working_group_staff,
+            RDRF_GROUPS.WORKING_GROUP_CURATOR: self.menu_config.working_group_curator,
+            RDRF_GROUPS.SUPER_USER: self.menu_config.super_user,
         }
         return switcher.get(group.lower(), [])
 
     def __init__(self, registries):
```

Comment:
`__init__` method should come before the other methods.

Reply:
Updated.